src/specs/models/GEM2SResponse.v1.yaml

```diff
@@ -1,22 +1,30 @@
-title: Pipeline response
+title: GEM2S response
```

Comment:
Can you check this schema again? It doesn't seem right. `experimentId` is type `string` but has `taskName` as it's attribute according to the indentation.